packages/eslint-plugin/src/rules/no-unsafe-token-fn-usage.test.ts

```diff
@@ -0,0 +1,38 @@
+import { tester } from 'src/test-utils'
+import rule, { RULE_NAME } from './no-unsafe-token-fn-usage'
+
+const imports = `import { css } from './panda/css'
+import { Circle } from './panda/jsx'
+`
+
+const valids = [
+  `const styles = css({ bg: 'token("colors.red.300") 50%' })`,
+  '<div className={css({ border: "solid {borderWidths.1} blue" })} />',
+]
+
+const invalids = [
+  'const styles = css({ bg: token("colors.red.300") })',
+  `const styles = css({ bg: 'token("colors.red.300")' })`,
+  '<div className={css({ border: "{borders.1}" })} />',
+  '<Circle _hover={{ color: "{colors.blue.300}" }} />',
+  '<Circle bg={token("colors.red.300")} />',
+  `<Circle bg={'token("colors.red.300")'} />`,
+  `<Circle bg='token("colors.red.300")' />`,
+]
```

Comment:
thinking about a different rule but a rule that checks for the validity of token paths (to prevent typos) is probably useful (we have something like that in the vscode ext)